test/integration/auth/rulesreview_test.go

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
Call this file `node_rule_resolution_test.go`.

Reply:
Ok.